src/main/scala/generate/database/PostgresGenerator.scala

```diff
@@ -7,30 +7,34 @@ import utils.StringUtils._
 object PostgresGenerator extends DatabaseGenerator {
 
   /** Given a column, parse it into the type required by Postgres */
-  private def parseColumn(column: Column): String =
+  private def parseColumnDef(column: ColumnDef): String =
     indent(
-      column match {
-        case IntColumn(name)        => s"$name INT"
-        case FloatColumn(name)      => s"$name REAL"
-        case StringColumn(name)     => s"$name TEXT"
-        case BoolColumn(name)       => s"$name BOOLEAN"
-        case DateColumn(name)       => s"$name DATE"
-        case TimeColumn(name)       => s"$name TIME"
-        case DateTimeTzColumn(name) => s"$name TIMESTAMPTZ"
-      },
-      length = 4
+      column.colType match {
+        case ColType.IntCol        => s"${column.name} INT"
+        case ColType.FloatCol      => s"${column.name} REAL"
+        case ColType.StringCol     => s"${column.name} TEXT"
+        case ColType.BoolCol       => s"${column.name} BOOLEAN"
+        case ColType.DateCol       => s"${column.name} DATE"
+        case ColType.TimeCol       => s"${column.name} TIME"
+        case ColType.DateTimeTzCol => s"${column.name} TIMESTAMPTZ"
```

Comment:
What about making a function that turns a `ColType` into a string, to save the replication of column.name?